inference-engine/src/plugin_api/cpp_interfaces/base/ie_executable_network_base.hpp

```diff
@@ -103,6 +104,10 @@ class ExecutableNetworkBase : public IExecutableNetwork {
     StatusCode GetContext(RemoteContext::Ptr& pContext, ResponseDesc* resp) const noexcept override {
         TO_STATUS(pContext = _impl->GetContext());
     }
+
+    std::shared_ptr<IExecutableNetworkInternal> GetImpl() const {
```

Comment:
should not we mark it as `deprecated`? I suppose it's used only in `ExecutableNetwork::reset` which is also deprecated

Reply:
The whole class should be marked as deprecated

Reply:
agree